src/polymer-cli.ts

```diff
@@ -122,7 +122,7 @@ export class PolymerCli {
     return Array.from(mergedArgs.values());
   }
 
-  run() {
+  run(): Promise<any> {
```

Comment:
When returning a promise we really want to use an async function to be sure that all returns and all errors are propagated consistently